lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -29,6 +29,16 @@ function getClientRect(element) {
   };
 }
 
+/**
+ * @param {CSSStyleDeclaration} style
+ * @return {boolean}
+ */
+function usesPixelArtScaling(style) {
+  return ['pixelated', 'crisp-edges'].includes(
```

Comment:
we might just want to do this inline, these get executed in browser which means we have to jump through some hoops like `/* istanbul ignore next */`, stringifying the function into global scope, etc

Reply:
Thanks for the tip. I had to inline it as otherwise the code would not run.

The only problem now is that there's duplicated code.